lib/ruby-debug-ide/commands/expression_info.rb

```diff
@@ -9,13 +9,20 @@ def regexp
     end
 
     def execute
-      string_to_parse = @match.post_match.gsub("\\n", "\n") + "\n\n\n"
+      string_to_parse = Command.unescape_incoming(@match.post_match) + "\n\n\n"
       total_lines = string_to_parse.count("\n") + 1
 
       lexer = RubyLex.new
       io = StringIO.new(string_to_parse)
+
+      begin
```

Comment:
I'd replace code from L18 till L30 with
set_io_encoding(io, string_to_parse.encoding) if string_to_parse.respond_to?(:encoding)
this way the code will be a little bit more clear.
What do you think?